internal/appsec/listener/httpsec/http.go

```diff
@@ -106,9 +107,9 @@ func (l *wafEventListener) onEvent(op *types.Operation, args types.HandlerOperat
 		// see if the associated user should be blocked. Since we don't control the execution flow in this case
 		// (SetUser is SDK), we delegate the responsibility of interrupting the handler to the user.
 		dyngo.On(op, func(operation *sharedsec.UserIDOperation, args sharedsec.UserIDOperationArgs) {
-			wafResult := shared.RunWAF(wafCtx, waf.RunAddressData{Persistent: map[string]any{UserIDAddr: args.UserID}}, l.config.WAFTimeout)
+			wafResult := shared.RunWAF(wafCtx, waf.RunAddressData{Persistent: map[string]any{UserIDAddr: args.UserID}})
 			if wafResult.HasActions() || wafResult.HasEvents() {
-				processHTTPSDKAction(operation, l.actions, wafResult.Actions)
+				shared.ProcessActions(operation, wafResult.Actions, types.NewMonitoringError("Request blocked"))
```

Comment:
so we don't propagate the status code of the blocked request in the error anymore? Do we still need all the data stored in the error?

Reply:
I see in the MonitoringError struct that you store the status code in it. Can we remove it?